tests/TestClassDatabaseChanges.php

```diff
@@ -70,6 +70,21 @@ public function prepare()
 
 		$this->_pWpOption = new WPOptionWrapperTest();
 		$this->_pDbChanges = new DatabaseChanges($this->_pWpOption, $wpdb);
+
+		add_option('onoffice-default-view', '');
```

Comment:
instead of using `add_option()` and `update_option()`, `$this->_pWpOption` can be used. That's especially handy for writing tests because it can be mocked.